modules/apps/digital-signature/digital-signature-web/src/main/resources/META-INF/resources/collect_digital_signature/view.jsp

```diff
@@ -31,6 +31,8 @@ if (digitalSignatureTitle != null) {
 		module="js/pages/CollectDigitalSignature"
 		props='<%=
 			HashMapBuilder.<String, Object>put(
+				"allowedFileExtension", StringUtil.merge(DigitalSignatureConstants.ALLOWED_FILE_EXTENSIONS)
```

Comment:
Follow the same name of constant (plural): 
**"allowedFileExtensions", StringUtil.merge(DigitalSignatureConstants.ALLOWED_FILE_EXTENSIONS**